django_netjsonconfig/base/template.py

```diff
@@ -53,6 +64,33 @@ class AbstractTemplate(BaseConfig):
                                                 'be automatically managed behind the scenes '
                                                 'for each configuration using this template, '
                                                 'valid only for the VPN type'))
+    flag = models.CharField(_('Flag'),
+                            choices=FLAG_CHOICES,
+                            default='private',
+                            max_length=16,
+                            db_index=True,
+                            help_text=_('Whether to keep this template private or make it '
+                                        'public, shared secretly or imported'))
```

Comment:
Let's change this to:

`Whether to keep this template private, share it publicly, share it privately using a secret key or import its contents from another source`